lib/quby/answers/services/answer_validator.rb

```diff
@@ -85,6 +104,23 @@ def validate_float(question, validation, value)
           answer.send(:add_error, question, :valid_float, validation[:message] || "Invalid float")
         end
 
+        def validate_date(question, validation, value)
+          # Skip this validation if all date parts are empty
+          return if value.values.all?(&:blank?)
+
+          send_date_error(question, validation) if value.values.any?(&:blank?)
+
+          begin
+            convert_answer_value(question, value)
```

Comment:
Is hier een test voor? 
heb namelijk het gevoel dat value hier een hash is en dat convert_answer_value een string verwacht.